src/microbe_stage/Microbe.Contact.cs

```diff
@@ -505,14 +505,19 @@ public void Kill()
             }
         }
 
-        int chunksToSpawn = Math.Max(1, HexCount / Constants.CORPSE_CHUNK_DIVISOR);
+        // Queues either 1 corpse chunk or a factor of the organelles
+        int chunksToSpawn = Math.Max(1, organelles.Count / Constants.CORPSE_CHUNK_DIVISOR);
```

Comment:
As this uses organelle count now, CORPSE_CHUNK_DIVISOR should be updated to reflect, probably should make that half as big as it is now, to more (roughly) match the existing functionalites.